src/Saml2/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
-        "lightsaml/lightsaml": "^2.0",
+         "litesaml/lightsaml": "^3.0",
```

Comment:
```suggestion
        "litesaml/lightsaml": "^3.0",
```

Reply:
Oh dear! Not sure how did I do that! Thanks! @lucasmichot